docker-compose.yml

```diff
@@ -0,0 +1,14 @@
+services:
+  mkdocs:
+    image: python:3-alpine
+    volumes:
+      - ./:/docs
+    stdin_open: true
+    tty: true
+    working_dir: /docs
+    # This could be more efficient, but it works.
+    command: 'sh -c "apk add bash git && pip install --no-cache-dir -r requirements.txt && mkdocs serve -a 0.0.0.0:80"'
```

Comment:
I wonder if you considered using [a prebuilt mkdocs image](https://hub.docker.com/search?q=mkdocs).

There does not seem to be an official one, but building on existing work might be helpful.

Reply:
The first I tried died on missing the `mkdocs-multirepo-plugin`. The next I looked at talked so much about "syncing your documentation" that I got cold feet. By then I'd wasted so much time that I needed to get moving.

It's worth revisiting at some time, but currently I'm apparently the only one that felt the need, so... :smiley: